src/staff_augmentation/components/GetFreeQuote/GetFreeQuoteContent/FreeQuote/QuotesForm/index.js

```diff
@@ -40,15 +39,7 @@ function QuotesForm ({ initialValues, onSubmitFinished, ...props }) {
     }
 
     try {
-      await axios.post(MAILER_URL, data)
-    } catch (_) {
-      window.alert('An error occurred while sending your message.\n\nPlease contact us at info@wedevelop.me')
-
-      return undefined
-    }
-
-    try {
-      await axios.post(STAFF_AUGMENTATION_FORM_PROCESSOR_URL, values)
+      await axios.post(STAFF_AUGMENTATION_FORM_PROCESSOR_URL, values, data)
```

Comment:
Here, you don't have to send the `data` variable to the cloud function. In fact, passing this variable as the third parameter to `axios.post` will be used as an Axios configuration object and not data to be sent.

The data to be sent on the body of the request is the second parameter (`values`). That's why on the FrontEnd you can access the form field values with `values.email` and `values.message`, but in the backend as `req.body.email` and `req.body.message`. You're sending these values here as the request body.

You have to cut and paste the definition of the `data` variable from here to the cloud function, and then use it to call `sendEmail(data)`. You'll also need to move the constant that indicates the `to` email address from the FrontEnd's constants to the cloud function's constants.

```suggestion
      await axios.post(STAFF_AUGMENTATION_FORM_PROCESSOR_URL, values)
```